ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -92,6 +92,8 @@ object LedgerApiTestTool {
         sys.exit(1)
       })
 
+    Thread.sleep(5000)
```

Comment:
Should be replaced by either exponential back-off or ensuring that the participants are online before the test tool runs. I'd favor the latter.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/2737/commits/5a733e99406ca835afe746d1ad39421bd23faefd